### PR TITLE
PMM-9722 upgrade clickhouse version

### PR DESCRIPTION
**Pull request by nikita-b**

### CODE REVIEW

**Review by denisok — APPROVED** [review]

File: ansible/playbook/tasks/roles/clickhouse/defaults/main.yml [hunk 1]
```diff
@@ -1 +1 @@
-clickhouse_version: 21.3.14.1-2
+clickhouse_version: 21.3.20.1-2
```
[the remark this answers is not in the document]
Comment (nikita-b) [reply ansible/playbook/tasks/roles/clickhouse/defaults/main.yml:1]: @denisok I had a discussion with the Clickhouse team and this version also has fix

https://kb.altinity.com/upgrade/vulnerabilities/
I'll send commit to Slack

[the remark this answers is not in the document]
Comment (atymchuk) [reply ansible/playbook/tasks/roles/clickhouse/defaults/main.yml:1]: Why not upgrading to 21.8.15.7-lts?

**Review by BupycHuk — APPROVED** [review]

File: ansible/playbook/tasks/roles/clickhouse/defaults/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikita-b) [reply ansible/playbook/tasks/roles/clickhouse/defaults/main.yml:1]: @atymchuk we need only fix CVE here. Fewer changes => fewer possibilities of incompatibilities :)

**Comment by denisok** [comment]
couple of other places where we mention clickhouse versions or images:
https://github.com/percona/qan-api2/blob/main/Makefile#L40
https://github.com/percona/qan-api2/blob/main/docker-compose.yaml#L16
https://github.com/percona/grafana-dashboards/blob/main/pmm-app/docker-compose.yaml#L16

**Pull request merged**